src/OnlineRetailPortal.Contracts/Contracts/ICategoryResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts.Contracts
+{
+    public interface ICategoryResponse
```

Comment:
why interface?

Reply:
for failure and success response, but as we discuss to go with middleware , i'll remove it

Reply:
i remove the interface as there is no failure response